package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "slimerjs-core",
+  "version": "1.0.0",
+  "description": "A scriptable browser like PhantomJS, based on Firefox",
+  "bin": {
+    "slimerjs-firefox": "./bin/slimerjs-core"
+  },
+  "main": "./lib/slimerjs-core.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/laurentj/slimerjs.git"
+  },
+  "scripts": {
+    "install": "node install.js"
```

Comment:
The only reason we run install.js is to populate lib/location.js, which defines the path of the binary using `which`. Don't we still want to do that?